tests/cephadm/test_cephadm_log_spam.py

```diff
@@ -0,0 +1,29 @@
+from cli.exceptions import OperationFailedError
+
+
+def run(ceph_cluster, **kw):
+    """Verify spam logs not present in cephadm.log
+    Args:
+        **kw: Key/value pairs of configuration information
+              to be used in the test.
+    """
+    node = ceph_cluster.get_nodes(role="_admin")[0]
+    CEPHADM_LOG_PATH = "/var/log/ceph/cephadm.log"
+
+    client = node.rssh if kw.get("sudo", False) else node.ssh
```

Comment:
Why this check, where we are passing `sudo` to `kw` ?

Reply:
Missed this, will change with next patch

Reply:
Done